lib/adal-angular.js

```diff
@@ -338,7 +338,9 @@ if (typeof module !== 'undefined' && module.exports) {
                         if (tokenStored) {
                             authService.info('Token is avaliable for this url ' + config.url);
                             // check endpoint mapping if provided
-                            config.headers.Authorization = 'Bearer ' + tokenStored;
+                            if(config.headers.Authorization){
```

Comment:
I think you want the reverse check i.e. when `config.headers.Authorization` is not set, then set it, otherwise not.